deployment/charts/templates/fuel-core-deploy.yaml

```diff
@@ -21,6 +21,31 @@ data:
   chainspec.json: |-
 {{ .Files.Get "chainspec.json" | indent 4 }}
 ---
+{{- if .Values.app.sentry_enabled }}
+kind: Service
+apiVersion: v1
+metadata:
+  labels:
+    app: {{ template "fuel-core.name" . }}
+    chart: {{ template "fuel-core.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+  name: {{ template "fuel-core.name" . }}-lb-service
+spec:
+  type: LoadBalancer
+  selector:
+    app: {{ template "fuel-core.name" . }}
+  ports:
+    - name: http
+      port: {{ .Values.app.http_port }}
+      protocol: TCP
+      targetPort: {{ .Values.app.target_port }}
+    - name: p2p
+      port: {{ .Values.app.peering_port }}
+      protocol: TCP
+      targetPort: {{ .Values.app.peering_port }}
```

Comment:
```suggestion
```

We will support exposing p2p publically after we have peer reputation for block sync: https://github.com/FuelLabs/fuel-core/issues/943

Reply:
thanks for removing this